src/BaseHandlerTrait.php

```diff
@@ -22,13 +23,18 @@ trait BaseHandlerTrait
     /**
      * @return Decoder
      *
-     * @deprecated will be removed in 5.0 - as of v3 of webauthn-lib the Decoder is now created within both:
+     * @deprecated 4.5.0 Will be removed without equivalent functionality to replace it
+     *
+     * No longer needed as of v3 of webauthn-lib the Decoder is now created within both:
      * AttestationObjectLoader::__construct()
      * AuthenticatorAssertionResponseValidator::__construct()
      */
     protected function getDecoder(): Decoder
     {
-        return new Decoder(new TagObjectManager(), new OtherObjectManager());
+        return Deprecation::withNoReplacement(function () {
```

Comment:
Because of this method call, we need to add framework as an explicit dependency with a `^4.12` constraint.

Reply:
Updated